w3c.json

```diff
@@ -0,0 +1,5 @@
+ {
+    "group":      109735
+,   "contacts":   ["dontcallmedom", "himorin"]
+,   "repo-type":  "note"
```

Comment:
is the intent really to publish this as a WG Note? My sense is that it's more of a CG item, and probably more in the "tool" or "other" repo-type.

Reply:
changed to "tool"